src/frontend/components/SøknadsSteg/OmDeg/useOmdeg.tsx

```diff
@@ -50,6 +51,7 @@ export const useOmdeg = (): {
         if (skalTriggeEøsForSøker(oppdatertSøker) !== søkerTriggerEøs) {
             settSøkerTriggerEøs(prevState => !prevState);
         }
+        validerAlleSynligeFelter();
```

Comment:
Finnes det noen bedre måte å gjøre dette på? gjøres for å aktivere neste knapp når noe kommer fra en delvis utfylt state.